pybliometrics/scopus/utils/get_content.py

```diff
@@ -163,17 +172,21 @@ def get_folder(api, view):
     from configparser import NoOptionError
     from pathlib import Path
 
-    from pybliometrics.scopus.utils import CONFIG_FILE, DEFAULT_PATHS
-    from pybliometrics.scopus.utils.create_config import create_config
+    from pybliometrics.scopus.utils import DEFAULT_PATHS
+
+    config = get_config()
+    config_path = get_config_path()
 
     if not config.has_section('Directories'):
-        create_config()
+        error_message = 'No configuration file found, please create one by initializing the pybliometrics library '\
+            'with init()'
+        raise FileNotFoundError(error_message)
```

Comment:
This is a validity test for the config file. I think we should move this to the `init()`, but in a slighly adopted version: No error message, let's just test whether the three sections Directories, Keys, Requests exist. If not, let's throw a NoSectionError. That, we simply try to access these sections in a loop and let the configparser throw this exceptionl